x-pack/plugins/infra/public/common/visualizations/lens/utils.ts

```diff
@@ -70,7 +70,7 @@ export const getXYVisualizationState = (
     showSingleSeries: false,
   },
   valueLabels: 'show',
-  fittingFunction: 'None',
+  fittingFunction: 'Zero',
```

Comment:
I added it here because it should be applied to all charts, this can change if we need different settings per chart